src/main/webapp/i18n/en/team.json

```diff
@@ -68,9 +71,78 @@
                 "label": "Delete",
                 "error": "Could not remove team. Please try again."
             },
-            "errors": {
-                "studentAlreadyAssignedToTeam": "Student with login {{ studentLogin }} is already assigned to team with id {{ teamId }."
-            }
+            "importTeams": {
+                "buttonLabel": "Import teams",
+                "dialogTitle": "Import teams into:",
+                "introText": "This dialog can be used to import teams from another exercise into this exercise.",
```

Comment:
I don't know if its necessary, but maybe we want to add that the source exercise has to be in the same course to prevent questions.

Reply:
Good point. I'm not sure if users would assume that you can import teams from a different course since different courses also have different students usually but I agree that being as explicit as possible is best. I'll add it.